internal/usecase/repository/domain_repository.go

```diff
@@ -422,35 +391,11 @@ func (r *domainRepository) updateIpaDomain(
 		return err
 	}
 
-	if err = db.Omit(clause.Associations).
+	if err = db.
 		Create(data).
 		Error; err != nil {
 		return err
 	}
 
-	// CaCerts
```

Comment:
Does GORM add new certs/servers/locations, update existing entries, and remove entries that are no longer present?

Reply:
GORM adds all the certs/servers/locations.  But, for whatever reason (I couldn't figure it out) it doesn't remove or update existing entries.  So you end up with duplicates / superseded records.  However, look up ~10 lines and you will see that we `Delete` all the old records before `Create`ing the new ones.

Reply:
Oh? That sounds horribly inefficient.

Reply:
It's exactly what was happening before, but accomplished with fewer lines of code now :)

Yeah, it's not ideal.  But it works - I won't lose sleep over it!